packages/widget/src/components/Card/CardButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { Button } from '@mui/material';
+import { alpha, styled } from '@mui/material/styles';
+
+export const CardButton = styled(Button)(({ theme }) => {
```

Comment:
This will cause a conflict I think with the CardButton that is currently on the Bookmark PR. I can change the name on other PR if you like will connect with you tomorrow :-)